src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search__form {
+  position: relative;
+  background-color: #fff;
+  margin-top: 20px;
+}
+
+.search__form--big {
+  height: 70px;
+}
+
+.search__form--small {
+  height: 42px;
+}
+
+.search__icon--big {
+  width: 19px;
+  height: 19px;
+}
+
+.search__icon--small {
+  width: 11px;
+  height: 11px;
+}
+
+.search__icon-container {
+  position: absolute;
+  line-height: 0;
+}
+
+.search__icon-container--big {
+  left: 26px;
+  top: 25px;
+}
+
+.search__icon-container--small {
+  left: 13px;
+  top: 15px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search__input--big {
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search__input--small {
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search__input::placeholder {
+  color: #3d4e61;
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search__input:focus {
+  outline: none;
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
The font-family 'Avenir-Heavy' is not defined as a separate font-family in the @font-face declarations. Ensure that 'Avenir-Heavy' is correctly defined or use 'Avenir' with the appropriate font-weight to achieve the desired effect.